TypeCobol.LanguageServer/TypeCobolServer.cs

```diff
@@ -431,10 +432,10 @@ protected override void OnDidSaveTextDocument(DidSaveTextDocumentParams paramete
         {
             if (parameters.text != null)
             {
-                DidChangeTextDocumentParams dctdp = new DidChangeTextDocumentParams(parameters.textDocument.uri);
+                var vtdi = new VersionedTextDocumentIdentifier(parameters.textDocument.uri, 0);
```

Comment:
using `null` for version seems more appropriate as our server does not handle this field yet. Which value is sent by the client on a DidChangeText ?

Reply:
The client aways sends 0 for version.

Reply:
Ok we can keep 0 here too then